### ISSUE TITLE
Can't fully install dependencies after fresh download - expo-yarn-workspace postinstall fails

### ISSUE THREAD

**Issue by patsa** [sysgears/apollo-universal-starter-kit#1196]
**Describe the bug**
I just downloaded the current Master of the repo and can't install the dependencies fully because of an issue with the execution of the `postinstall` (`expo-yarn-workspace postinstall`) script within the mobile package.

While using Node 12.16.1 I get the follwing error:

```
error C:\Users\wasd\Downloads\apollo-universal-starter-kit-master\node_modules\mobile: Command failed.
Exit code: 1
Command: expo-yarn-workspaces postinstall
Arguments:
Directory: C:\Users\wasd\Downloads\apollo-universal-starter-kit-master\node_modules\mobile
Output:
internal/validators.js:117
    throw new ERR_INVALID_ARG_TYPE(name, 'string', value);
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received null
    at validateString (internal/validators.js:117:11)
    at Object.join (path.js:375:7)
    at symlinkNecessaryPackage (C:\Users\wasd\Downloads\apollo-universal-starter-kit-master\node_modules\expo-yarn-workspaces\bin\symlink-necessary-packages.js:39:35)
    at symlinkNecessaryPackages (C:\Users\wasd\Downloads\apollo-universal-starter-kit-master\node_modules\expo-yarn-workspaces\bin\symlink-necessary-packages.js:17:3)
    at Object.<anonymous> (C:\Users\wasd\Downloads\apollo-universal-starter-kit-master\node_modules\expo-yarn-workspaces\bin\symlink-necessary-packages.js:81:3)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
```

While using Node 10.19.0 I get the follwing error:

```
error C:\Users\wasd\Downloads\apollo-universal-starter-kit-master\node_modules\mobile: Command failed.
Exit code: 1
Command: expo-yarn-workspaces postinstall
Arguments:
Directory: C:\Users\wasd\Downloads\apollo-universal-starter-kit-master\node_modules\mobile
Output:
internal/validators.js:125
    throw new ERR_INVALID_ARG_TYPE(name, 'string', value);
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type object
    at validateString (internal/validators.js:125:11)
    at Object.join (path.js:427:7)
    at symlinkNecessaryPackage (C:\Users\wasd\Downloads\apollo-universal-starter-kit-master\node_modules\expo-yarn-workspaces\bin\symlink-necessary-packages.js:39:35)
    at symlinkNecessaryPackages (C:\Users\wasd\Downloads\apollo-universal-starter-kit-master\node_modules\expo-yarn-workspaces\bin\symlink-necessary-packages.js:17:3)
    at Object.<anonymous> (C:\Users\wasd\Downloads\apollo-universal-starter-kit-master\node_modules\expo-yarn-workspaces\bin\symlink-necessary-packages.js:81:3)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
```

While using Node 8.17.0 I get the follwing error:

```
error C:\Users\wasd\Downloads\apollo-universal-starter-kit-master\node_modules\mobile: Command failed.
Exit code: 1
Command: expo-yarn-workspaces postinstall
Arguments:
Directory: C:\Users\wasd\Downloads\apollo-universal-starter-kit-master\node_modules\mobile
Output:
path.js:28
    throw new TypeError('Path must be a string. Received ' + inspect(path));
    ^

TypeError: Path must be a string. Received null
    at assertPath (path.js:28:11)
    at Object.join (path.js:489:7)
    at symlinkNecessaryPackage (C:\Users\wasd\Downloads\apollo-universal-starter-kit-master\node_modules\expo-yarn-workspaces\bin\symlink-necessary-packages.js:39:35)
    at symlinkNecessaryPackages (C:\Users\wasd\Downloads\apollo-universal-starter-kit-master\node_modules\expo-yarn-workspaces\bin\symlink-necessary-packages.js:17:3)
    at Object.<anonymous> (C:\Users\wasd\Downloads\apollo-universal-starter-kit-master\node_modules\expo-yarn-workspaces\bin\symlink-necessary-packages.js:81:3)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
```

**To Reproduce**
1. Download the repository.
2. cd into the downloaded directory
3. execute `yarn`

**Expected behavior**
Installation of dependencies succeeds with all postinstall steps.

**Desktop (please complete the following information):**
 - OS: Win 10 64
 - Version [master]

**Comment by patsa** [comment]
By the way, I use yarn `1.21.1`.

**Comment by lokeshrana9999** [comment]
@patsa although not a full-proof solution, deleting the yarn.lock file and executing yarn works for me, you can try that.

**Comment by nomi9995** [comment]
@patsa can you share root package.json workplaces array or object? I think problem is there

**Comment by larixer** [comment]
Fixed via #1220
